gasp-node/rollup/node/src/chain_spec.rs

```diff
@@ -243,6 +243,183 @@ pub fn rollup_local_config(
 	)
 }
 
+pub fn holesky_testnet(decode_url: Option<String>) -> ChainSpec {
+	let eth_chain_id = 17000u64;
+	let chain_genesis_salt_arr: [u8; 32] =
+		hex_literal::hex!("0022002200220022002200220022002200220022002200220022002200220022");
+
+	let collator01 = hex_literal::hex!("65Cd3e834B975038F462D64696dfBA275B35ADF4");
+	let collator01_sr25519 =
+		hex_literal::hex!("240970b59630cc9f2f02fe84de116e0ddc6c52a17e8702c8b7625330612b076f");
+	let collator01_ed25519 =
+		hex_literal::hex!("af9b01d518e34fbc25514bb37734cd2a8e77059523a9f6c8c8e81c131405b9aa");
+
+	let collator02 = hex_literal::hex!("2Bb0c51238a267ca0Ee6e895D8DbfBE9b351cCA4");
+	let collator02_sr25519 =
+		hex_literal::hex!("d05fad2d6fc4cb2b2f66008a377b0c0a72313606e9d90e0818e55074b97e2b54");
+	let collator02_ed25519 =
+		hex_literal::hex!("31f0c35e2305ee7ccfcdb71b5a55f80191f40cc86350f5ff31cdb38712e23857");
+
+	let sudo = hex_literal::hex!("E5C44a37d87BF55295E39CE1fbD0177A390d4376");
+
+	// Give your base currency a unit name and decimal places
+	let mut properties = sc_chain_spec::Properties::new();
+	properties.insert("tokenSymbol".into(), "GASP".into());
+	properties.insert("tokenDecimals".into(), 18u32.into());
+	properties.insert("ss58Format".into(), 42u32.into());
+	properties.insert("isEthereum".into(), true.into());
+	// This is quite useless here :/
+	properties.insert(
+		"chainGenesisSalt".into(),
+		array_bytes::bytes2hex("0x", chain_genesis_salt_arr).into(),
+	);
+
+	let decode_url = decode_url.expect("polkadot url is provided");
+	// todo builder
+	ChainSpec::from_genesis(
+		// Name
+		"Holesky-testnet",
+		// ID
+		"holesky",
+		ChainType::Live,
```

Comment:
Maybe, you'd want to change the chaiinType here so that we can edit the types on polkadot apps ui on the fly